test/neutronics/test_neutron_source.py

```diff
@@ -0,0 +1,8 @@
+from libra_toolbox.neutronics.neutron_source import *
+from collections.abc import Iterable
+
+def test_get_avg_neutron_rate(tmpdir):
+
+    source = A325_generator_diamond((0,0,0), (0,0,1))
+
+    assert source == Iterable[openmc.Source]
```

Comment:
This cannot work as:

1. `openmc.Source` is not a type
2. type comparison is not done with `==` with with `isinstance`

Consider doing:

```suggestion
def test_get_avg_neutron_rate():

    source = A325_generator_diamond((0, 0, 0), (0, 0, 1))

    assert isinstance(source, Iterable)
    for s in source:
        assert isinstance(s, openmc.IndependentSource)
```